content/flutter-publishing/publishing-to-app-store.md

```diff
@@ -66,21 +66,30 @@ Once the Apple Developer Portal has been enabled for the account or team the app
 2. Click **App Store Connect**.
 3. If you have several keys available, select the right key in the **App Store Connect API key** field.
 4. Mark the **Publish even if tests fail** checkbox to continue uploading the app artifact even when the tests failed.
-4. Mark the **Submit to TestFlight beta review** checkbox to submit the build for beta review and prepare it for distributing to beta testers. Note: This action is performed during [post-processing](#post-processing-of-app-store-connect-distribution).
-5. Mark the **Distribute to beta groups** checkbox and enter the names of the beta groups to automatically distribute the build to the testers in those groups once the build has passed beta review. Note: This action is performed during [post-processing](#post-processing-of-app-store-connect-distribution).
-6. Select **Enable App Store Connect publishing** at the top of the section to enable publishing.
+5. Select **Enable App Store Connect publishing** at the top of the section to enable publishing.
```

Comment:
The section header should read "Step 3. Enabling App Store Connect publishing for workflow"